### PR TITLE
Bug/refactor colors #278

### LINKED ISSUE #278: Refactor colors
There is a styles namespace (`status-im.components.styles`) and there are colors in this namespace. However, it is still possible to see styles which look that way:

`(def scan-text
  {:flex          1
   :flexDirection :column
   :color         "#7099e6"
   :margin-left   8})`

It is, of course, not important, but we should spend some time replacing colors with pre-defined colors from `status-im.components.styles`

### PR DESCRIPTION
**Pull request by mintheredonethat**
fixes #278

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,6 +37,9 @@ status-im.iml
 node_modules/
 npm-debug.log
 
+env/dev/env/android/main.cljs
+.re-natal
```
Comment (jarradh) [inline .gitignore]: can't ignore this file and shouldn't

**Comment by jarradh** [comment]
Thanks for your contribution @mintheredonethat ! I added "fixes #278" to your PR comment, because github will auto-close the issue when we accept the merge. 

We only accept PR's that have their changes squashed into a single commit.

What's your rational for ignoring these files? How are they generated from scratch?

**Comment by mintheredonethat** [comment]
Hey @jarradh, glad to be finally contributing after having all those build issues.

Yeah, I probably should have just not staged those files for commitment. I'll submit another pull request with the proper settings?

**Comment by jarradh** [comment]
you don't need to make a new PR, you can squash and push to your branch and PR will update

**Comment by alwx** [comment]
Hey, @mintheredonethat 
Thanks for the PR!

There are some colors in `status.components.styles` namespace that we don't use and don't need (like `text4-color` and `text5-color`, probably there are more). I think it would be nice to remove these colors from the file in order to make it more readable.

**Comment by mintheredonethat** [comment]
Hey, @alwx - It seems like it was only text4-color

Let me know of further changes!

**Comment by alwx** [comment]
@mintheredonethat as I see, we import `text5-color`, but don't use it in our code.

**Comment by mintheredonethat** [comment]
@alwx good catch - would you like me to also comb through the project for other instances of importing without using?

**Comment by alwx** [comment]
@mintheredonethat yes, if it is not a big problem and doesn't take much time

**Pull request merged**